### PR TITLE
Consolidating logging auth failures into a single place

### PR DESCRIPTION
**Pull request by aklish**

### CODE REVIEW

File: elide-core/src/main/java/com/yahoo/elide/Elide.java [hunk 1]
```diff
@@ -111,7 +114,11 @@ public ElideResponse get(
             auditLogger.commit();
             transaction.commit();
             requestScope.runCommitTriggers();
+            logSecurityExceptions(requestScope);
             return response;
+        } catch (ForbiddenAccessException e) {
```
Comment (wcekan) [inline elide-core/src/main/java/com/yahoo/elide/Elide.java]: Should we handle this case once in buildErrorResponse

File: elide-core/src/main/java/com/yahoo/elide/core/exceptions/ForbiddenAccessException.java [hunk 2]
```diff
@@ -15,8 +16,15 @@
 public class ForbiddenAccessException extends HttpStatusException {
     private static final long serialVersionUID = 1L;
 
-    public ForbiddenAccessException(String verboseMessage) {
+    private RequestScope scope;
```
Comment (wcekan) [inline elide-core/src/main/java/com/yahoo/elide/core/exceptions/ForbiddenAccessException.java]: Should be transient

**Comment by wcekan** [comment]
:+1: No problem.  We can clean up in future PR.

**Comment by yahoocla** [comment]
CLA is valid!

**Pull request merged**